src/docbuilder/rustwide_builder.rs

```diff
@@ -424,6 +436,11 @@ impl RustwideBuilder {
         build_dir.purge()?;
         krate.purge_from_cache(&self.workspace)?;
         local_storage.close()?;
+        if let Some(image) = metadata.docker_image() {
+            std::process::Command::new("docker")
```

Comment:
```suggestion
            if image != "rustops/crates-build-env" {
                std::process::Command::new("docker");
            }
```
Otherwise someone could DOS the queue by publishing a bunch of crates with crates-build-env set.

Actually, I guess they could do that anyway, crates-build-env isn't special ... we just use it by default, so it will be redownloaded. But if you publish a bunch of crates in a row with the same image, it will redownload the image for each crate.

@pietroalbini what do you think, is that a threat model worth worrying about? It won't *break* the server, it will just make builds really slow.

Reply:
Some other ways to break this:
- Make an enormous (like 100gb) docker image
- Make small images which can download quickly enough to hit docker's new ratelimits